res/values/strings.xml

```diff
@@ -157,6 +157,7 @@
     <string name="switch_camera">Switch Camera</string>
     <string name="toggle_fullscreen">Toggle Full Screen Mode</string>
     <string name="location">Location</string>
+    <string name="location">Locations</string>
```

Comment:
Is this by accident - two strings `location`, one of them in singular, one in plural?

Reply:
thanks, of course, this is an error, i'll fix that